src/fio/FioEngine.ts

```diff
@@ -198,6 +198,18 @@ export class FioEngine extends CurrencyEngine<FioTools, SafeFioWalletInfo> {
       fioAction: async (actionName: string, params: any): Promise<any> => {
         return await this.multicastServers(actionName, params)
       },
+      fetchFioAddresses: async (): Promise<FioAddress[]> => {
+        await this.refreshFioAddresses()
```

Comment:
Since we are doing the fetch right away, should we delete the `try/catch` block from the `refreshFioAddresses` method? That way, if you call `fetchFioAddresses`, and something goes wrong, you get a normal error instead of a `console.warn` and an empty return value.